### DISCUSSION TITLE
Open Local File automatically in VSCode Web

### DISCUSSION THREAD

**Discussion by danijel-prodanovic** [microsoft/vscode-discussions#546] [Extension Development QnA]
We have a notebook extension, initially built for **VSCode Desktop**. Now, when launching a notebook editor from the web page, we don't want to require the user to install VSCode just to be able to edit some notes. So if a VSCode is not locally installed, we would like to open the editor in the browser by using the **VSCode for Web** instead.
Although there is a nice possibility in VSCode Desktop to initialize with the certain file open, by using vscode://file/{path}, I wonder if there is a mechanism that would make it possible to load **vscode.dev** with the certain file opened automatically.

**Comment by isidorn** [comment]
Can you clarify this ask? Is this still needed?

**Comment by burkeholland** [comment]
Closing as this is more than 30 days old. If this is still an issue, please feel free to re-open.
